ClosedXML/Utils/OpenXmlHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DocumentFormat.OpenXml;
+
+namespace ClosedXML.Utils
+{
+    internal static class OpenXmlHelper
```

Comment:
Converting between OpenXML and ClosedXML variables should be limited to loading and saving. I think they should remain private members of the `XLWorkbook` class.

Reply:
I also need this in the XCLFBaseConverter. I tend to follow DRY principles and see no problem with extracting this method into a utility class. Would you rather a different approach here?

Reply:
Yes, I agree now.